tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import tempfile
+
+import pytest
+
+
+@pytest.fixture()
+def tempdir():
+    with tempfile.TemporaryDirectory() as tempdir:
+        yield tempdir
```

Comment:
pytest has a built-in fixture for this:
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Didn't know about that, thanks!